### PR TITLE
Tab completion documentation

### PR DESCRIPTION
**Pull request by jburger**
This PR introduces documentation for tab completion in Octo CLI

Related PR https://github.com/OctopusDeploy/OctopusCLI/pull/53

### CODE REVIEW

**Comment by jburger** [comment]
@wordlee when you get a moment, could you cast your eye over the build errors, have you seen one like that before?

**Comment by wordlee** [comment]
> @wordlee when you get a moment, could you cast your eye over the build errors, have you seen one like that before?

It might be the Gif. I'll comment it out to see if that's it.

**Comment by wordlee** [comment]
The docsync agent has run out of disc space. Should hopefully be back up and running soon.

**Comment by jburger** [comment]
Ah righto - thanks!

**Comment by wordlee** [comment]
I added a couple of line edits (mostly capitalization and removed extra words). I also combined the code samples so that they'll be combined into one block with tabs that let readers choose the code sample they need.

File: docs/octopus-rest-api/octopus-cli/index.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ octo ... --server http://your-octopus-server/ --apiKey API-ABCDEF123456
 Most commands also support [JSON formatted output](formatted-output.md).
 
 :::success
-**Creating API keys**
+## Creating API keys {#OctopusCLI-CreatingAPIKeys}
```
Comment (wordlee) [inline docs/octopus-rest-api/octopus-cli/index.md]: Minor tweak
```suggestion
## Create an API key {#OctopusCLI-CreatingAPIKeys}
```

File: docs/octopus-rest-api/octopus-cli/index.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ octo ... --server http://your-octopus-server/ --apiKey API-ABCDEF123456
 Most commands also support [JSON formatted output](formatted-output.md).
 
 :::success
-**Creating API keys**
+## Create an API key {#OctopusCLI-CreatingAPIKeys}
```
Comment (wordlee) [inline docs/octopus-rest-api/octopus-cli/index.md]:
```suggestion
**Create an API key**
```

File: docs/octopus-rest-api/octopus-cli/index.md [hunk 3]
```diff
@@ -80,7 +80,7 @@ octo ... --server http://your-octopus-server/ --apiKey API-ABCDEF123456
 Most commands also support [JSON formatted output](formatted-output.md).
 
 :::success
```
Comment (wordlee) [inline docs/octopus-rest-api/octopus-cli/index.md]:
```suggestion

```

File: docs/octopus-rest-api/octopus-cli/index.md [hunk 4]
```diff
@@ -80,7 +80,7 @@ octo ... --server http://your-octopus-server/ --apiKey API-ABCDEF123456
...
-**Creating API keys**
+## Create an API key {#OctopusCLI-CreatingAPIKeys}
 Learn about [how to create an API key](/docs/octopus-rest-api/how-to-create-an-api-key.md).
 :::
```
Comment (wordlee) [inline docs/octopus-rest-api/octopus-cli/index.md]:
```suggestion

```

File: docs/octopus-rest-api/octopus-cli/index.md [hunk 5]
```diff
@@ -79,13 +79,57 @@ octo ... --server http://your-octopus-server/ --apiKey API-ABCDEF123456
 
 Most commands also support [JSON formatted output](formatted-output.md).
 
-:::success
-**Creating API keys**
+**Create an API key**
```
Comment (wordlee) [inline docs/octopus-rest-api/octopus-cli/index.md]: Ugh. 
```suggestion
:::success
**Create an API key**
:::
```

**Pull request merged**